README.txt

```diff
@@ -27,39 +27,47 @@ This game is not finished
 - Don't expect it to work as well as a finished game will.
 - Please report any bugs. When doing that, debug.txt is useful.
 
-Default Controls
+Default controls
 -----------------
-- WASD: move
-- Space: jump/climb
-- Shift: sneak/go down
-- Q: drop itemstack (+ SHIFT for single item)
-- I: inventory
-- Mouse: turn/look
-- Mouse left: dig/punch
-- Mouse right: place/use
-- Mouse wheel: select item
-- T: chat
-- 1-8: select item
-
-- Esc: pause menu (pauses only singleplayer game)
-- R: Enable/Disable full range view
+- Move mouse: Look around
+- W, A, S, D: Move
+- Space: Jump/move up
+- Shift: Sneak/move down
+- Q: Drop itemstack
+- Shift + Q: Drop single item
+- Left mouse button: Mine/punch/take item
+- Right mouse button: Build/use
+- Shift + right mouse button: Build (without using)
+- I: Inventory menu
+- Mouse wheel: Select item
+- 0-9: Select item
+- Z: Zoom (needs zoom privilege)
+- T: Chat
+- /: Commad
+
+- Esc: Pause menu/abort/exit (pauses only singleplayer game)
```

Comment:
Esc doesn't cause abort or exit directly, so i think 'Pause and access exit menu'.